test/test.ts-snapshots/dark-browser.html

```diff
@@ -0,0 +1,2 @@
class="edgeLabel"><g class="label" transform="translate(0,0)"><rect height="0" width="0" ry="0" rx="0"></rect><foreignObject height="0" width="0"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml"><span style="" class="edgeLabel L-LS-A&#x27; L-LE-C" id="L-L-A-C"></span></div></foreignObject></g></g><g style="opacity: 1;" transform="" class="edgeLabel"><g class="label" transform="translate(0,0)"><rect height="0" width="0" ry="0" rx="0"></rect><foreignObject height="0" width="0"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml"><span style="" class="edgeLabel L-LS-B&#x27; L-LE-D" id="L-L-B-D"></span></div></foreignObject></g></g><g style="opacity: 1;" transform="" class="edgeLabel"><g class="label" transform="translate(0,0)"><rect height="0" width="0" ry="0" rx="0"></rect><foreignObject height="0" width="0"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml"><span style="" class="edgeLabel L-LS-C&#x27; L-LE-D" id="L-L-C-D"></span></div></foreignObject></g></g></g><g class="nodes"><g style="opacity: 1;" transform="translate(62.19140625,27.5)" id="flowchart-A-40" class="node default"><rect class="label-container" height="39" width="29.4375" y="-19.5" x="-14.71875" ry="0" rx="0"></rect><g transform="translate(0,0)" class="label"><g transform="translate(-4.71875,-9.5)"><foreignObject height="19" width="9.4375"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml">A</div></foreignObject></g></g></g><g style="opacity: 1;" transform="translate(22.53125,116.5)" id="flowchart-B-41" class="node default"><rect class="label-container" height="39" width="29.0625" y="-19.5" x="-14.53125" ry="0" rx="0"></rect><g transform="translate(0,0)" class="label"><g transform="translate(-4.53125,-9.5)"><foreignObject height="19" width="9.0625"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml">B</div></foreignObject></g></g></g><g style="opacity: 1;" transform="translate(101.8515625,116.5)" id="flowchart-C-43" class="node default"><rect class="label-container" height="39" width="29.578125" y="-19.5" x="-14.7890625" ry="0" rx="0"></rect><g transform="translate(0,0)" class="label"><g transform="translate(-4.7890625,-9.5)"><foreignObject height="19" width="9.578125"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml">C</div></foreignObject></g></g></g><g style="opacity: 1;" transform="translate(62.19140625,205.5)" id="flowchart-D-45" class="node default"><rect class="label-container" height="39" width="29.8125" y="-19.5" x="-14.90625" ry="0" rx="0"></rect><g transform="translate(0,0)" class="label"><g transform="translate(-4.90625,-9.5)"><foreignObject height="19" width="9.8125"><div style="display: inline-block; white-space: nowrap;" xmlns="http://www.w3.org/1999/xhtml">D</div></foreignObject></g></g></g></g></g></g></svg></p>
```

Comment:
How often does mermaid change its styles?
Will these snapshots break with each update?

Reply:
I’m not afraid Mermaid (or svgo) will change snapshots between updates, because this project has a lockfile.

The snapshots are inconsistent between my machine and CI though, and sometimes even between runs. This probably has the same cause as #5 and #6. I’m planning to look into this next.